http/server.go

```diff
@@ -88,55 +76,27 @@ func (server *APIServer) StartUnsecured(edgeMode bool) error {
 		EdgeManager:          server.edgeManager,
 		KubeClient:           server.kubeClient,
 		KubernetesDeployer:   server.kubernetesDeployer,
-		Secured:              false,
+		Secured:              !edgeMode,
 		ContainerPlatform:    server.containerPlatform,
 	}
 
-	var h http.Handler = handler.NewHandler(config)
+	var httpHandler http.Handler = handler.NewHandler(config)
```

Comment:
I know it was like this, but looks better IMO 
```suggestion
	httpHandler := handler.NewHandler(config)
```